thrust/testing/cuda/partition.cu

```diff
@@ -677,9 +677,12 @@ void TestPartitionIfWithMagnitude(int magnitude)
 void TestPartitionIfWithLargeNumberOfItems()
 {
   TestPartitionIfWithMagnitude(30);
+  // These require 64-bit dispatches even when magnitude < 32.
+#  ifndef THRUST_FORCE_32_BIT_OFFSET_TYPE
```

Comment:
Do we typically indent macros? Most in this PR are not indented, but there are a few exceptions.

Reply:
Our clang-format did that.

Reply:
Note that it isn't indenting to the surrounding code level, but because this `#ifndef` is inside of another `#if`.

Reply:
Gotcha, thanks for the explanation!